### ISSUE TITLE
Check all of the RPC endpoint related transaction

### ISSUE THREAD

**Issue by shuse2** [LiskHQ/lisk-sdk#4327]
### Description

After #4325 and #3980, it needs to make sure all the endpoints are returning and receiving
expected values.

#### Affected RPCs
- postBlock
- blocks
- getTransactions
- postTransaction

### Motivation

It should make sure that all the endpoint are updated.

### Acceptance Criteria

- All the tests related to RPC are updated
- All the endpoints are returning updated schema

**Comment by shuse2** [comment]
Superseded by #4326
